website/docs/usage/projects.md

```diff
@@ -148,6 +148,13 @@ skipped. You can also set `--force` to force re-running a command, or `--dry` to
 perform a "dry run" and see what would happen (without actually running the
 script).
 
+By default `spacy projects run` checks your installed dependencies to verify
+that your environment is properly set up and aligns with the project's
+`requirements.txt`, if there is one. If missing or conflicting dependencies are
+detected, a corresponding warning is displayed. If you'd like to disable the
+dependency check, set `check_requirements: false` in your project's
+`project.yml`.
```

Comment:
```suggestion
Since spaCy v3.4.2, `spacy projects run` checks your installed dependencies to verify
that your environment is properly set up and aligns with the project's
`requirements.txt`, if there is one. If missing or conflicting dependencies are
detected, a corresponding warning is displayed. If you'd like to disable the
dependency check, set `check_requirements: false` in your project's
`project.yml`.
```

Reply:
This comment is good, but we should also update https://spacy.io/usage/projects#project-yml: the example project should have 
```
check_requirements: true
```
in it's `project.yml`, and the option needs to be added to the table with a note that it's new since 3.4.2.

Reply:
> ...but we should also update https://spacy.io/usage/projects#project-yml: the example project should have...

https://github.com/explosion/projects/pull/131